packages/core/src/dropdown/Dropdown.css

```diff
@@ -22,6 +21,7 @@
 
 .saltDropdown:hover {
   background: var(--dropdown-background-hover);
+  cursor: var(--salt-cursor-hover);
```

Comment:
I don't think these types of changes were necessary but since they've been done it's fine to keep